components/arm/fake/fake.go

```diff
@@ -74,17 +75,17 @@ func NewArm(cfg config.Component, logger golog.Logger) (arm.LocalArm, error) {
 		}
 
 		switch converted.ArmModel {
-		case xarm.ModelName6DOF:
+		case xarm.ModelName6DOF.String():
```

Comment:
Good catch, I think I misunderstand where the value was coming from. I assumed since it was from converted attributes, it'd using the JSON unmarshalling for Model where it'd automatically prepend "rdk:builtin" for configs with only the short model name. I think I see an easy fix though. `string(xarm.ModelName6DOF.Name)` and similar should do it (compare only against the short name)